about.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: About Us
+---
+
+#  <span style="color:#EB090D">Divided by COVID. Connected by Python</span>
+
+## What is PyCon? 
+
+

```

Comment:
nit: any reason for the double empty lines?

Reply:
yeah, as you can see in https://in.pycon.org/blog/2020/2020-call-for-volunteers.html, I feel there's not enough space between the headings and the content. So, that's why I've used double lines for the headings.

Reply:
So, should I keep the double lines, or make them single line?

Reply:
Yeah. That makes sense.

Overall, I feel the paragraph spacing needs to be fixed. Although this is an unrelated change.

Reply:
@Sc0rpi0n101 Please keep them for now. We will review later.